packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -3132,16 +3132,23 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
         // We want to send in points that are centered around a (0,0) origin, so
         // we cache the position.
         _pointOffsetOrigin = point.offset;
+        final TextPosition currentTextPosition = TextPosition(
```

Comment:
Ideally we should avoid using the selection value from the controller. The value from the controller is potentially one frame ahead of the currently rendered selection range.

Reply:
Then we have to update the floating cursor after the frame